From 5105fedd7fa13dadd2d0d864fb77873b83b79a4b Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@dominion.thruhere.net>
Date: Thu, 23 Jun 2011 13:52:09 +0200
Subject: [PATCH] buildsys: hardcode socketdir to /tmp

the TMPDIR env var isn't always pointing to the right target path

Upstream-Status: Inappropriate [embedded]

Signed-off-by: Koen Kooi <koen@dominion.thruhere.net>

Original comment:

	avoid to check tmp dir at build time. instead uses hard coded /tmp here
	comment added by Kevin Tian <kevin.tian@intel.com>
---
 configure.ac |   11 +----------
 1 files changed, 1 insertions(+), 10 deletions(-)

diff --git a/configure.ac b/configure.ac
index 408054b..6d26180 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1483,16 +1483,7 @@ AC_SUBST(TEST_LAUNCH_HELPER_BINARY)
 AC_DEFINE_UNQUOTED(DBUS_TEST_LAUNCH_HELPER_BINARY, "$TEST_LAUNCH_HELPER_BINARY",
                    [Full path to the launch helper test program in the builddir])
 
-#### Find socket directories
-if ! test -z "$TMPDIR" ; then
-   DEFAULT_SOCKET_DIR=$TMPDIR
-elif ! test -z "$TEMP" ; then
-   DEFAULT_SOCKET_DIR=$TEMP
-elif ! test -z "$TMP" ; then
-   DEFAULT_SOCKET_DIR=$TMP
-else
-   DEFAULT_SOCKET_DIR=/tmp
-fi
+DEFAULT_SOCKET_DIR=/tmp
 
 DEFAULT_SOCKET_DIR=`echo $DEFAULT_SOCKET_DIR | sed 's/+/%2B/g'`
 
-- 
1.6.6.1

